### DISCUSSION TITLE
Custom renovated version

### DISCUSSION THREAD

**Discussion by jorfermo** [renovatebot/renovate#31536] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

GitHub - renovate renovate@38.93.3

### Please tell us more about your question or problem

I'm using only the major version to reference a terraform module like this:

```terraform
module "apps" {
  source = "github.com/myorg/terraform-local-input-reader.git?ref=1"
}
```

We set it up to ignore minor and patch versions, but when renovate suggests an upgrade to the next major version, it's done like this:

```terraform
module "apps" {
  - source = "github.com/myorg/terraform-local-input-reader.git?ref=1"
  + source = "github.com/myorg/terraform-local-input-reader.git?ref=2.0.3"
}
```

Is there any way to set the renovate version to be only the next major? Like this:

```terraform
module "apps" {
  - source = "github.com/myorg/terraform-local-input-reader.git?ref=1"
  + source = "github.com/myorg/terraform-local-input-reader.git?ref=2"
}
```

### Logs (if relevant)

<details><summary>Logs</summary>

```
```

</details>

**Comment by rarkins** [comment, marked as answer]
Try setting versioning=docker for this package. Docker versioning ensures that "precision" is maintained, e.g. 1->2, 1.1->2.0, etc.
